feature-pack/src/main/resources/modules/system/layers/base/org/jboss/resteasy/resteasy-jaxrs/main/module.xml

```diff
@@ -48,7 +48,7 @@
         <module name="org.jboss.resteasy.resteasy-validator-provider" optional="true" services="export" export="true"/>
         <module name="org.jboss.logging"/>
         <module name="org.reactivestreams"/>
-        <module name="org.eclipse.microprofile.restclient"/>
+        <module name="org.eclipse.microprofile.restclient" export="true"/>
```

Comment:
@ronsigal Please approve this change.

@ehsavoie why is exporting this module necessary, versus some other mechanism of making it available to whatever code is getting it via the export?

Reply:
I need it for the rest-client-*tck part but the proper way I think is to add it as a dependency of the deployment.